<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QWidget {
        background-color: #121212;
        color: white;
        font-family: 'Segoe UI';
    }
    QLabel {
        color: #ffffff;
    }
    QPushButton {
        background-color: #1e1e1e;
        color: white;
        border-radius: 10px;
        padding: 8px;
    }
    QPushButton#createAccountButton {
        background-color: #f44336;
        font-weight: bold;
        color: white;
    }
    QLineEdit {
        background-color: #2a2a2a;
        border: none;
        border-radius: 10px;
        padding: 10px;
        color: white;
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
     </font>
    </property>
    <property name="text">
     <string>Chon 1 mon an</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_bundau">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
     </font>
    </property>
    <property name="text">
     <string>Bún đậu mắm tôm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_bunbohue">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
     </font>
    </property>
    <property name="text">
     <string>Bún bò Huế</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_phobo">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>330</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
     </font>
    </property>
    <property name="text">
     <string>Phở Bò</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>340</y>
      <width>281</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../image-removebg-preview (2) (1).png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>221</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/—Pngtree—bun dau mam tom art_15322189 (2).png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>130</y>
      <width>291</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/pngtree-delicious-beef-vietnamese-rice-noodles-png-image_2878222-removebg-preview.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>370</y>
      <width>301</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/ad5dfcfd926c92feba8aa281dbad3a53_t__1_-removebg-preview.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
